Symfony/CS/Tests/Fixer/PSR2/EofEndingFixerTest.php

```diff
@@ -52,17 +52,46 @@ public function provideCases()
 ',
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;",
+                "<?php\r\$a = 4;\n",
+                "<?php\r\$a = 4;",
             ),
             array(
-                "<?php\r\$a = 1;\n",
-                "<?php\r\$a = 1;\r",
+                "<?php\r\$a = 5;\n",
+                "<?php\r\$a = 5;\r",
```

Comment:
I agree, as stated. 

With this PR I try to stop the fixer from adding a line break in a trailing `T_INLINE_HTML` (which is named IMO) block at the end of a (PHP) file, no matter if there are Mac, Windows or other characters in there. As a side effect it solves the issue addressed by @gharlan in https://github.com/FriendsOfPHP/PHP-CS-Fixer/pull/1193#discussion_r30738637.

However; this PR is not about  issue https://github.com/FriendsOfPHP/PHP-CS-Fixer/issues/1092, but it doesn't introduce it either nor makes it worse. So I suggest discussing that issue and possible solution there.